### PR TITLE
Clarify 'non-exhaustive list of the values defined by this specification'

### LINKED ISSUE #138: "A non-exhaustive list of valid values defined by this specification are" is weird language
The current draft contains phrases about "non-exhaustive list of valid values defined by this specification" in a few places:

>  non-exhaustive list of valid values defined by this specification are string, number, and image media types such as image/jpeg as defined in IANA media type registry for images 

(e.g. https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#appendix-E.2.2 )

> To express the specifics about the claim, the most deeply nested value MAY be an object that includes a following non-exhaustive list of parameters defined by this specification:

(e.g.  https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#appendix-E.1.1.2 )

In both cases I've not been able to find an exhaustive list of the values that the specification does actually define. I'm not sure exactly what is meant, in other parts of the specification language like "Below is a non-exhaustive list of valid parameters that MAY be included" is used which may be the intention, or perhaps it is more like "This specification defines the following parameters, other parameters may be used and implementations MUST ignore unrecognised parameters"?

### PR DESCRIPTION
**Pull request by jogu**
closes #138

### CODE REVIEW

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (selfissued) [inline openid-4-verifiable-credential-issuance-1_0.md:1877]:
```suggestion
  * `credentialSubject`: OPTIONAL. Object containing a list of name/value pairs, where each name identifies a claim offered in the Credential. The value can be another such object (nested data structures), or an array of such objects. To express the specifics about the claim, the most deeply nested value MAY be an object that includes the following parameters. Other parameters MAY also be used.
```

Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:1877]: please add `the following parameters defined by this specification. Other...`

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1879]:
```suggestion
      * `value_type`: OPTIONAL. String value determining the type of value of the claim. Valid values are `string`, `number`, and image media types such as `image/jpeg` as defined in IANA media type registry for images (https://www.iana.org/assignments/media-types/media-types.xhtml#image). Other values MAY also be used.
```

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:1940]: please add `the following parameters defined by this specification. Other...`

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1942]:
```suggestion
      * `value_type`: OPTIONAL. String value determining the type of value of the claim. Valid values are `string`, `number`, and image media types such as `image/jpeg` as defined in IANA media type registry for images (https://www.iana.org/assignments/media-types/media-types.xhtml#image). Other values MAY also be used.
```

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:2033]:
```suggestion
    * `value_type`: OPTIONAL. String value determining the type of value of the claim. Valid values are `string`, `number`, and image media types such as `image/jpeg` as defined in IANA media type registry for images (https://www.iana.org/assignments/media-types/media-types.xhtml#image). Other values MAY also be used.
```

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:2083]:
```suggestion
* `claims`: OPTIONAL. Object containing a list of name/value pairs, where each name identifies a claim about the subject offered in the Credential. The value can be another such object (nested data structures), or an array of such objects. To express the specifics about the claim, the most deeply nested value MAY be an object that includes the following parameters defined by this specification:
```

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:2085]:
```suggestion
    * `value_type`: OPTIONAL. String value determining the type of value of the claim. Valid values are `string`, `number`, and image media types such as `image/jpeg` as defined in IANA media type registry for images (https://www.iana.org/assignments/media-types/media-types.xhtml#image). Other values MAY also be used.
```

**Review by Sakurann — CHANGES_REQUESTED** [review]
the current language conveys two messages:
1. this spec defines valid values
2. other values can be used.
If the language is being changed, please include point 1 - I made suggestions.

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (jogu) [reply openid-4-verifiable-credential-issuance-1_0.md:1877]: @selfissued @Sakurann  Are we definitely okay removing the ':'? This is the start of a list and everywhere else in the spec we use a colon in this way.

Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:1877]: oh, umm let's keep `:` probably.

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (jogu) [reply openid-4-verifiable-credential-issuance-1_0.md:1940]:
```suggestion
  * `credentialSubject`: OPTIONAL. Object containing a list of name/value pairs, where each name identifies a claim offered in the Credential. The value can be another such object (nested data structures), or an array of such objects. To express the specifics about the claim, the most deeply nested value MAY be an object that includes the following parameters defined by this specification (other parameters MAY also be included):
```

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (jogu) [reply openid-4-verifiable-credential-issuance-1_0.md:1877]:
```suggestion
  * `credentialSubject`: OPTIONAL. Object containing a list of name/value pairs, where each name identifies a claim offered in the Credential. The value can be another such object (nested data structures), or an array of such objects. To express the specifics about the claim, the most deeply nested value MAY be an object that includes the following parameters defined by this specification (other parameters MAY also be used):
```

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (jogu) [inline openid-4-verifiable-credential-issuance-1_0.md:1879]:
```suggestion
      * `value_type`: OPTIONAL. String value determining the type of value of the claim. Valid values defined by this specification are `string`, `number`, and image media types such as `image/jpeg` as defined in IANA media type registry for images (https://www.iana.org/assignments/media-types/media-types.xhtml#image). Other values MAY also be used.
```

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (jogu) [inline openid-4-verifiable-credential-issuance-1_0.md:1942]:
```suggestion
      * `value_type`: OPTIONAL. String value determining the type of value of the claim. Valid values defined by this specification are `string`, `number`, and image media types such as `image/jpeg` as defined in IANA media type registry for images (https://www.iana.org/assignments/media-types/media-types.xhtml#image). Other values MAY also be used.
```

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (jogu) [inline openid-4-verifiable-credential-issuance-1_0.md:2033]:
```suggestion
    * `value_type`: OPTIONAL. String value determining the type of value of the claim. Valid values defined by this specification are `string`, `number`, and image media types such as `image/jpeg` as defined in IANA media type registry for images (https://www.iana.org/assignments/media-types/media-types.xhtml#image). Other values MAY also be used.
```

File: openid-4-verifiable-credential-issuance-1_0.md
Comment (jogu) [inline openid-4-verifiable-credential-issuance-1_0.md:2085]:
```suggestion
    * `value_type`: OPTIONAL. String value determining the type of value of the claim. Valid values defined by this specification are `string`, `number`, and image media types such as `image/jpeg` as defined in IANA media type registry for images (https://www.iana.org/assignments/media-types/media-types.xhtml#image). Other values MAY also be used.
```

**Comment by jogu** [comment]
I've put back all the 'defined by this specification'. As per discussion I've not made the changes that required removed the ':'s. This should be good to go now.

**Review by Sakurann — APPROVED** [review]

**Comment by Sakurann** [comment]
lgtm. will merge once 1-2 more approvals kick in.

**Review by bc-pi — APPROVED** [review]

**Pull request merged**
